src/org/zaproxy/zap/extension/authstats/ExtensionAuthStats.java

```diff
@@ -157,4 +132,36 @@ public void onHttpResponseReceive(HttpMessage msg, int initiator, HttpSender sen
             log.error(e.getMessage(), e);
         }
     }
+
+    private static void updateAuthIndicatorStats(HttpMessage msg, String site, String prefix, Context context) {
+        AuthenticationMethod auth = context.getAuthenticationMethod();
+        if (auth == null) {
+            // No authentication method no indicators.
```

Comment:
The second part of the comment should be removed....or really I think the code is readable the comment could probably be removed.

Reply:
OK, I'll go for the latter then.